gcloud/credentials.py

```diff
@@ -239,10 +236,8 @@ def _get_service_account_name(credentials):
              account type.
     """
     service_account_name = None
-    if isinstance(credentials, client.SignedJwtAssertionCredentials):
-        service_account_name = credentials.service_account_name
-    elif isinstance(credentials, service_account._ServiceAccountCredentials):
-        service_account_name = credentials._service_account_email
+    if isinstance(credentials, ServiceAccountCredentials):
```

Comment:
It's just a sign of how poorly implemented service accounts were before. Still work to be done.